### ISSUE TITLE
Android Auto only work if i'm on "Park"

### ISSUE THREAD

**Issue by rafoloru** [gartnera/headunit#152]
Hello, 
My AA v1.11 works only if i'm in "Parking" mode in my car. As soon as I put "Drive" Android Auto disapear

**Comment by Trevelopment** [comment]
Are you shifting into drive slow enough that maybe you trigger the backup camera for just a second?

**Comment by rafoloru** [comment]
Hi,

There are some development in this case.


   1. Android Auto (from All in One tools) was installed in my car and
   everythings worked perfectly
   2. MZD-AIO have upgraded to 2.8.4-1 and AA to 1.11. I installed it, and
   after that AA never wanted to open.
   3. I tried a lot of things here (Uninstall and reinstall AA. Tried with
   another phone etc... nothing working)
   4. I upgraded the firmware of my car to v55.00.760 and reinstall AA
   5. Magic AA was working but only in Park mode
   6. The next day AA was working perfectly in Drive mode without any
   change (yeah)
   7. Everythings was working perfectly during 2 or 3 weeks and one morning
   AA won't start anymore (no change have been made, no new tweak installation
   etc. The day after it was working and next day he won't
   8. Now i'm stuck exactly in the same situation as number 2. My next
   steop is to update again teh firmware, but I don,t want to do that each
   month haha
